src/common/types/club.ts

```diff
@@ -6,6 +6,7 @@ export interface Member {
   email: string;
   name: string;
   image: string;
+  assetId?: string;
```

Comment:
What’s assetId

Reply:
`assetId` is the id of the user's profile image. I am storing it so that we can delete the old profile picture when the user uploads a new one.

Reply:
Can we rename it so it’s clear it’s for the user’s image?

Reply:
Sure. I've updated to `imageId` to match the `image` field in the user object